src/main/kotlin/com/markskelton/notification/Notifications.kt

```diff
@@ -0,0 +1,56 @@
+package com.markskelton.notification
+
+import com.intellij.ide.plugins.PluginManagerCore.getPlugin
+import com.intellij.ide.plugins.PluginManagerCore.getPluginOrPlatformByClassName
+import com.intellij.notification.Notification
+import com.intellij.notification.NotificationAction
+import com.intellij.notification.NotificationDisplayType
+import com.intellij.notification.NotificationGroup
+import com.intellij.notification.NotificationListener
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.options.ShowSettingsUtil
+import com.markskelton.settings.ThemeSettingsUI
+
+val UPDATE_MESSAGE: String = """
+      What's New?<br>
+      <ul>
+        <li>Theme is now configurable through settings.</li>
+      </ul>
+      <br>Please see the <a href="https://github.com/one-dark/jetbrains-one-dark-theme/blob/master/CHANGELOG.md">Changelog</a> for more details.
+      <br>
+      Thank you for choosing the One Dark Theme!<br>
+""".trimIndent()
+
+const val CURRENT_VERSION = "4.0.0"
```

Comment:
Any way to pull this from the theme without having to hard code it?

Reply:
Yep! I can add that.